tests/test_immutableset.py

```diff
@@ -269,3 +269,10 @@ def test_pickling(self):
         self.assertEqual(immutableset([5, 2]).__reduce__(), (immutableset, ((5, 2),)))
         self.assertEqual(immutableset([5]).__reduce__(), (immutableset, ((5,),)))
         self.assertEqual(immutableset().__reduce__(), (immutableset, ((),)))
+
+    def test_subtract_from_normal_set(self):
+        self.assertEqual({5, 6}, frozenset([4, 5, 6]) - immutableset([2, 3, 4]))
```

Comment:
Can you separate the `frozenset` examples from the `set` examples by putting them at the bottom and grouping them with a newline in between?